mlfinlab/codependence/information_matrix.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
@proskurin do we need a separate file for this code? Should this code form part of the codependence module, will users make use of these two functions in areas other than just clustrered feature importance? 

I think we need to provide a lot more documentation to show how these functions are used. If they are just for feature clustring importance then maybe they should be hidden?

Reply:
We should make a separate file for matrix calculations, the users can use it for a separate research purposes